VERSION

```diff
@@ -1 +1 @@
-0.90.0.0
\ No newline at end of file
+1.0.0.0-rc1
```

Comment:
the big question is, do we want to use the tweak level?

Reply:
IMHO, nope. Why would it be needed?